### PR TITLE
Add looks :fire:

### PR DESCRIPTION
**Pull request by sopheakneak**
:fire: :fire: :fire:
:fire_engine:

### CODE REVIEW

**Comment by sopheakneak** [comment]
Yes, there are a lot of `<br />` haha, but they're intentional. I wanted to place the text in the middle.

**Comment by sopheakneak** [comment]
:1st_place_medal:

File: src/PreGraduationComponent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    render() {
+        return (
+            <div className = 'test'>
+                <CSSTransitionGroup transitionName="main" transitionEnter={false} transitionLeave={false} transitionAppear={true}
+                    transitionAppearTimeout={1000}>
+                    <header className='jumbo center'>
+                        <br />
+                        <br />
+                        <br />
+                        <div>Let's Check Out Those Universities Graduation Rate</div>
```
Comment (zmbc) [inline src/PreGraduationComponent.js]: "Universities' Graduation Rates"

File: src/PreMapComponent.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    render() {
+        return (
+            <div className = 'test'>
+                <CSSTransitionGroup transitionName="main" transitionEnter={false} transitionLeave={false} transitionAppear={true}
+                    transitionAppearTimeout={1000}>
+                    <header className='jumbo center'>
+                        <br />
+                        <br />
+                        <div>
+                    <span>For your filters, there are over <Count maxNumber={this.props.data.length} duration='8' styleClass='emphasis' textInfo='Universities' /></span>
```
Comment (zmbc) [inline src/PreMapComponent.js]: I know this was in the original too, but I'd remove the word "over"

File: src/PreMapComponent.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      e.preventDefault();
+      hashHistory.push('/map');
+    }
+    render() {
+        return (
+            <div className = 'test'>
+                <CSSTransitionGroup transitionName="main" transitionEnter={false} transitionLeave={false} transitionAppear={true}
+                    transitionAppearTimeout={1000}>
+                    <header className='jumbo center'>
+                        <br />
```
Comment (zmbc) [inline src/PreMapComponent.js]: Don't use br's

File: src/PreSelectivity.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    render() {
+        return (
+            <div className = 'test'>
+                <CSSTransitionGroup transitionName="main" transitionEnter={false} transitionLeave={false} transitionAppear={true}
+                    transitionAppearTimeout={1000}>
+                    <header className='jumbo center'>
+                        <br />
+                        <br />
+                        <br />
+                        <div>Let's Take a Look at How <span className='emphasis'>Selectivity</span> They Are</div>
```
Comment (zmbc) [inline src/PreSelectivity.js]: "Selectivity" -> "Selective"

**Review by zmbc — CHANGES_REQUESTED** [review]

**Comment by sopheakneak** [comment]
I tried to configure the flexbox but I had trouble with bootstrap CSS overwriting the style. I tried !important children div to flex but it doesn't work. I believe this is how display: flex works:
`<div style={{'display:flex}}>parent div is now flex so all children div is flex<div/>`
`<div>chidren div now got overwritten by boostrap {display:block} so it couldn't flex</div>` :muscle:

**Review by zmbc — APPROVED** [review]

**Pull request merged**